### ISSUE TITLE
My user primary key is in a different column

### ISSUE THREAD

**Issue by Intelemon** [bavix/laravel-wallet#635]
**Describe your task**
A clear and concise description of your task.
I am using this code to specify my User  primary key to use another column
```
public $incrementing = false;
protected $primaryKey = 'code';
```
I cannot seem to find how to specify which user column to use when creating a wallet. It defaults to my 'code' column which allows string. I want to use the users.id when creating wallet.

**Server:**
 - php version: 8.0
 - database: [e.g mysql 8.0]
 - wallet version [e.g. 8.2]

**Comment by rez1dent3** [comment]
@Intelemon hello. Unfortunately, the package will automatically take the key specified in pk. There is no configuration option here.

**Comment by rez1dent3** [comment]
I think that such a configuration will definitely not be in the near future. At least I don't plan to.

Why are you having such difficulty? What problem are you trying to solve? In fact, you can do something like [this](https://github.com/bavix/laravel-wallet-uuid) for your "code" and work like a pk.

**Comment by Intelemon** [comment]
While reading through the issues section, I found that in your latest version, the holder_id column in the wallet table accepts a string as an input type. I am currently using version 7.* on my Laravel 8 project which only allows an integer as an input type. This is an issue for me because the whole project relies on the user table's code column.  What I did is just remove the following code which makes the foreign key back to the users' table id column and then define in all relationships to use the users' table code column instead of the id column.
`
public $incrementing = false;
protected $primaryKey = 'code';
`
